app/assets/stylesheets/application.scss

```diff
@@ -28,3 +24,72 @@
     margin: 0;
   }
 }
+
+// Move down content because we have a fixed navbar that is 50px tall
+body {
+  padding-top: 50px;
+}
+
+//
+// Sidebar
+//
+
+// Hide for mobile, show later
+.sidebar {
+  display: none;
+}
+
+@media (min-width: 768px) {
+  .sidebar {
+    background-color: #f5f5f5;
+    border-right: 1px solid #eee;
+    bottom: 0;
+    display: block;
+    left: 0;
+    overflow-x: hidden;
+    overflow-y: auto; // Scrollable contents if viewport is shorter than content.
+    padding: 20px;
+    position: fixed;
+    top: 51px;
+    z-index: 1000;
+  }
+}
+
+// Sidebar navigation
+
+.nav-sidebar {
+  margin-bottom: 20px;
+  margin-left: -20px;
+  margin-right: -21px; // 20px padding + 1px border
+}
+
+.nav-sidebar > li > a {
+  padding-left: 20px;
+  padding-right: 20px;
+}
+
+.nav-sidebar > .active > a,
+.nav-sidebar > .active > a:hover,
+.nav-sidebar > .active > a:focus {
+  background-color: #428bca;
+  color: #fff;
```

Comment:
Color literals like `#fff` should only be used in variable declarations; they should be referred to via variable everywhere else.